### DISCUSSION TITLE
Sonoff Dongle USB Zigbee ZBDongle-E 3.0 and flash RCP firmware

### DISCUSSION THREAD

**Discussion by syntesys** [Koenkk/zigbee2mqtt#23286] [Devices]
Hello all,
I would enable Matter And Thread network on my ZBDongle-E 3.0. Now I'm using 7.4.2.0 firmware with ember driver, but I would flash the RCP firmware (same version). Do I have to re-pair all the devices after the flash of the new firmware or have I to change only the configuration of the adapter? Thank you.

**Comment by francisp2** [comment]
I don't think RCP firmware is still supported.

**Comment by syntesys** [comment]
I saw in darkxst github that the last version it's still compiled... Are you saying "supported by Z2M"?

**Reply by francisp2** [reply]
> "supported by Z2M

Indeed. But even Nabu Casa says you should not use multiprotocol any more, as they can't get it stable enough.

**Reply by Koenkk** [reply]
Indeed, do not use it

**Comment by syntesys** [comment]
Oh ok, thank you.
